packages/core/phase1/triggers/TRPR0002.test.ts

```diff
@@ -0,0 +1,54 @@
+import TriggerCode from "bichard7-next-data-latest/dist/types/TriggerCode"
+import generateAhoFromOffenceList from "../../phase2/tests/fixtures/helpers/generateAhoFromOffenceList"
+import type { Offence } from "../../types/AnnotatedHearingOutcome"
+import TRPR0002, { resultCodes } from "./TRPR0002"
+
+describe("TRPR0002", () => {
+  it.each(resultCodes)(
```

Comment:
I feel a bit uneasy about this - if resultCodes changes to something wrong then the test will still pass. Whilst duplication is generally not a good thing, in this case I think it's good because it's kind of like a double-check for what you're testing.

Reply:
yeah that makes sense. thanks for pointing it out